selfdrive/ui/SConscript

```diff
@@ -19,6 +19,8 @@ if arch == "Darwin":
   del base_libs[base_libs.index('OpenCL')]
   qt_env['FRAMEWORKS'] += ['OpenCL']
 
+qt_env['CXXFLAGS'] += ["-Wno-deprecated-declarations"]
```

Comment:
what's this for?

Reply:
This is for `ui` to compile in Qt 5.15. For example, one of the errors (without "-Wno-deprecated-declarations"):
```
./selfdrive/ui/qt/widgets/controls.h:226:61: error: 'buttonToggled' is deprecated: Use QButtonGroup::idToggled(int, bool) instead [-Werror,-Wdeprecated-declarations]
  226 |     QObject::connect(button_group, QOverload<int, bool>::of(&QButtonGroup::buttonToggled), [=](int id, bool checked) {
      |                                                             ^
/usr/include/aarch64-linux-gnu/qt5/QtWidgets/qbuttongroup.h:95:5: note: 'buttonToggled' has been explicitly marked deprecated here
   95 |     QT_DEPRECATED_VERSION_X_5_15("Use QButtonGroup::idToggled(int, bool) instead")
      |     ^
```
There are multiple errors like this. For example, [QButtonGroup::idToggled](https://doc.qt.io/qt-5/qbuttongroup.html#idToggled) was introduced in Qt 5.15.

We can replace them and remove "-Wno-deprecated-declarations" after we upgrade to Qt 5.15 LTS.